bazel/repositories.bzl

```diff
@@ -37,8 +37,8 @@ WINDOWS_SKIP_TARGETS = [
 NO_HTTP3_SKIP_TARGETS = [
     "envoy.quic.crypto_stream.server.quiche",
     "envoy.quic.deterministic_connection_id_generator",
-    "envoy.quic.crypto_stream.server.quiche",
```

Comment:
Can you comment what kinda extension needs to be listed here? Why envoy.quic.crypto_stream.server.quiche doesn't?

Reply:
It does. I'm not removing it. Its entry is duplicated.

Reply:
Woop! That's right.